static/source/css/global.styl

```diff
@@ -159,6 +159,9 @@ form
     .logo
         float: left
 
+        img
```

Comment:
This class is not used on the logo in `templates/global/menu.html` where the CSS class for the logo is set as `logo`. Since you changed the size of the logo, this will have breaking changes on the navbar unless you set the width to the `logo` class in this file for navbar or edit the logo in `templates/global/menu.html` to use both classes `img` and `logo` or set the width/height of the new image to the dimensions of the deleted file.